The order-only prerequisite operators (|) were removed since they cannot be
interpreted by Make 3.79.1 (which is used by Rtools 3.4). There were 3 instances
removed by this patch

diff --git a/src/libsass/Makefile b/src/libsass/Makefile
index 30023ba..8c4ed4d 100644
--- a/src/libsass/Makefile
+++ b/src/libsass/Makefile
@@ -204,17 +204,17 @@ debug-shared: shared
 lib:
 	$(MKDIR) lib
 
-lib/libsass.a: $(COBJECTS) $(OBJECTS) | lib
+lib/libsass.a: $(COBJECTS) $(OBJECTS) lib
 	$(AR) rcvs $@ $(COBJECTS) $(OBJECTS)
 
-lib/libsass.so: $(COBJECTS) $(OBJECTS) | lib
+lib/libsass.so: $(COBJECTS) $(OBJECTS) lib
 	$(CXX) -shared $(LDFLAGS) -o $@ $(COBJECTS) $(OBJECTS) $(LDLIBS)
 
-lib/libsass.dylib: $(COBJECTS) $(OBJECTS) | lib
+lib/libsass.dylib: $(COBJECTS) $(OBJECTS) lib
 	$(CXX) -shared $(LDFLAGS) -o $@ $(COBJECTS) $(OBJECTS) $(LDLIBS) \
 	-install_name @rpath/libsass.dylib
 
-lib/libsass.dll: $(COBJECTS) $(OBJECTS) $(RCOBJECTS) | lib
+lib/libsass.dll: $(COBJECTS) $(OBJECTS) $(RCOBJECTS) lib
 	$(CXX) -shared $(LDFLAGS) -o $@ $(COBJECTS) $(OBJECTS) $(RCOBJECTS) $(LDLIBS) \
 	-s -Wl,--subsystem,windows,--out-implib,lib/libsass.a
 
